app.rb

```diff
@@ -0,0 +1,59 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def create_student(age, name, parent_permission)
+    classroom = 'Null'
+    @people << { type: 'Student', data: Student.new(age, name, classroom, parent_permission) }
+  end
+
+  def create_teacher(age, name, spec)
+    @people << { type: 'Teacher', data: Teacher.new(age, spec, name) }
+  end
+
```

Comment:
- [ ] I am not able to create a `student` and `teacher` please address it
<img width="1440" alt="Screen Shot 2023-03-17 at 18 47 43" src="https://user-images.githubusercontent.com/44745855/225967748-5713bb93-f60f-49ca-a935-7ebe2a754cb6.png">

Reply:
Thanks. It has been fixed